content/release-10-17-0/devices-10-17-0.md

```diff
@@ -1,6 +1,198 @@
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 55%;"><col style="width: 15%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cloud Fieldbus</td>
+<td>The <b>Offset property</b> field in the register configuration now accepts all values.</td>
+<td>MTM-48256</td>
+<td>10.16.119.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Measurements</b> tab in the device details, a tooltip has been added which shows the whole values in case of truncated values in the Y-axis. Moreover, the information is provided that you can see the whole values by hovering over the axis.</td>
+<td>MTM-48603</td>
+<td>10.16.142.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>Devices can now send installed software information to the advanced-software-mgmt microservice without including the URL and software type. These parameters were incorrectly required before and are now made optional.</td>
+<td>DM-1431</td>
+<td>10.16.129.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>Fixed an issue where users could not find a location on the map based on the address they entered.</td>
+<td>MTM-49644</td>
+<td>10.16.104.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In the <b>Software repository</b> page, the <b>Add software</b> dialog accepted a software without software type although
+the field <b>Software type</b> is mandatory. This has been fixed and the <b>Add software</b> button is now active only when the
+mandatory <b>Software type</b> field is filled.</td>
+<td>DM-1713</td>
+<td>10.16.103.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In the <b>Software repository</b> page, sorting for the column "Device type" did not function properly. This issue is now fixed.</td>
+<td>DM-1712</td>
+<td>10.16.103.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In case of many active alarms of different severities, in the "Active critical alarms" widget users might have needed to press the <b>Load more</b> button several times on an empty list in order to see their active critical alarms. This is now fixed - the widget always initially shows active critical alarms if available and the <b>Load more</b> button does not load empty pages.</td>
+<td>DM-1646</td>
+<td>10.16.31.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Location</b> tab in the device details, you can now again change the location of the device on the map by dragging the position marker.</td>
+<td>DM-1565</td>
+<td>10.16.12.0</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>The device registration wizard had several layout and functionality issues which were fixed by aligning it to the default platform user experience.</td>
+<td>DM-1335</td>
+<td>10.16.8.0</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>- The LWM2M FOTA process now raises an alarm if the observation to the update stare resource of object 5 cannot be established.
```

Comment:
is "update stare resource" correct? specifically "stare" ?

Reply:
@sag-eweingaertner Can you please comment?

Reply:
Added a note to the ticket